src/OT.js

```diff
@@ -31,9 +31,18 @@ const checkAndroidPermissions = () => new Promise((resolve, reject) => {
 const setNativeEvents = (events) => {
   const eventNames = Object.keys(events);
   OT.setNativeEvents(eventNames);
-  each(events, (eventHandler, eventType) => {
+
+  // change in react native API : listeners does not exist any more with RN 0.64 but is replaced by listenerCount
+  let hasRegisteredEvents;
+  if (nativeEvents.listeners) {
     const allEvents = nativeEvents.listeners();
```

Comment:
`nativeEvents.listeners()` is the function that has been deprecated no?

Reply:
I added this for backward compatibility

Reply:
ok I think I get it, will test it this afternoon